lib/screens/more_details_page.dart

```diff
@@ -0,0 +1,223 @@
+        borderRadius: BorderRadius.circular(20.0),
+      ),
+      onPressed: () {
+        Navigator.push(
+          context,
+          MaterialPageRoute(builder: (context) => Home()), //archive question
+        );
+      },
+      child: Container(
+          child: Padding(
+        padding: const EdgeInsets.all(8.0),
+        child: Text("Archive this question"),
+      )),
+    );
+  }
+
+  Container _questionFieldViewer() {
+    return Container(
+      padding: const EdgeInsets.only(left: 20, top: 0, right: 20, bottom: 10),
+      height: MediaQuery.of(context).size.height - 405,
+      child: SingleChildScrollView(
+          child: Text(
+              "Long text here which is longer than the container heightLong text"
+              " here which is longer than the container heightLong text here"
+              " which is longer than the container heightLong text here which"
+              " is longer than the container heightLong text here which is "
+              "longer than the container heightLong text here which is longer"
+              " than the container heightLong text here which is longer than "
+              "the container heightLong text here which is longer than the contained"
+              " heightLong text here which is longer than the container height"
+              "Long text here which is longer than the container heightLong text"
+              " here which is longer than the container heightLong text here"
+              " which is longer than the container heightLong text here which"
+              " is longer than the container heightLong text here which is "
+              "longer than the container heightLong text here which is longer"
+              " than the container heightLong text here which is longer than "
+              "the container heightLong text here which is longer than the container"
+              " heightLong text here which is longer than the container height")),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: EurekaAppBar(
+        title: 'Question Details',
+        appBar: AppBar(),
+      ),
+      body: Column(
+        children: [
+          Container(
+            margin: EdgeInsets.all(20),
+            padding: EdgeInsets.only(left: 3, top: 0, right: 3, bottom: 5),
+            alignment: Alignment.topLeft,
+            width: double.infinity,
+            decoration: BoxDecoration(
+              color: Colors.grey[300],
+              borderRadius: BorderRadius.all(
+                Radius.circular(20.0),
+              ),
+            ),
+            child: Column(
+              children: [
+                _headerStack(),
+                Container(
+                  padding: EdgeInsets.all(5.0),
+                  child: Center(
+                    child: Column(
+                      children: <Widget>[
+                        _questionFieldViewer(),
+                      ],
```

Comment:
You don't need the Center here and you don't need this Column here since it's only one child, just call _questionFieldViewer() directly instead.

```suggestion
                Container(
                  padding: EdgeInsets.all(5.0),
                  child: _questionFieldViewer(),
                  ),
```

Reply:
done